src/index.ts

```diff
@@ -28,6 +30,21 @@ export interface FlashbotsOptions {
   revertingTxHashes?: Array<string>
 }
 
+/**
+ * This type was added by Lucas, in order to package all the information needed in a submittable bundle, as the
+ * {@link FlashbotsBundleProvider.publish} method
+ *
+ * It's fields are described as:
+ * signedBundledTransactions: an array of signed transaction that compose the bundle the user intends to submit
+ * blockTarget: the block number within which the bundle is valid and includable in a block
+ * options: an instance of FlashbotsOptions, used to specify extra information about the bundle's validity
+ */
+export interface FlashbotsBundle {
```

Comment:
is it really the best to use a FlashbotsBundle class for carrier txs? Blocknumber and options are unenforceable anyway (since the whole system works in a decentralized way)

could we not just pass in the values that we want? (since the rest of the ethers provider didn't create a bundle struct either)

Reply:
I guess I might have misunderstood what actually makes up a bundle: taking from the eth_sendBundle RPC these are the params it needs for a valid bundle submission.
If we were to remove these and only encode the signed transactions, I guess the bundle would be eternally valid and includable in a block by the validator: do you think this would be a better use of the carrier?